pkg/cluster/kubeadm/const.go

```diff
@@ -19,3 +19,6 @@ package kubeadm
 // APIServerPort is the expected default APIServerPort on the control plane node(s)
 // https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/#api-server-ports-and-ips
 const APIServerPort = 6443
+
+// Token defines a dummy, well known token for automating TLS bootstrap process
+const Token = "abcdef.0123456789abcdef"
```

Comment:
`DefaultToken`?

Reply:
I used the same convention used for `APIServerPort` (without default).